packages/amplify-cli/src/extensions/amplify-helpers/push-resources.ts

```diff
@@ -10,8 +10,14 @@ export async function pushResources(
   context: $TSContext,
   category?: string,
   resourceName?: string,
-  filteredResources?: { category: string, resourceName: string }[],
+  filteredResources?: { category: string; resourceName: string }[],
 ) {
+  if (context.parameters.options['iterative-rollback']) {
+    // validate --iterative-rollback with --force
+    if (context.parameters.options.force)
+      throw new Error("Consider runnning 'amplify push --force' to iteratively rollback and deploy new changes.");
```

Comment:
Can we make it clearer that --iterative-rollback and --force together will throw this error?

Something like:
"--iterative-rollback and --force cannot be used together. Consider runnning 'amplify push --force' to iteratively rollback and deploy new changes."